helm/alfresco-content-services/templates/deployment-repository.yaml

```diff
@@ -13,7 +13,6 @@ spec:
   strategy:
     {{- if eq .Values.repository.strategy.type "Recreate" }}
     type: {{ .Values.repository.strategy.type }}
-    rollingUpdate: null
```

Comment:
This was added for a reason to address a k8s issues (that granted may have been fixed by now). See https://issues.alfresco.com/jira/browse/MNT-21042 and https://github.com/Alfresco/acs-deployment/commit/09b41c0c2eddad5985238ceb931d564d881ff1c5 for more details.

Reply:
it's an invalid value I think the issue got it wrong, rollingUpdate section can be present only when type is RollingUpdate not Recreate

Reply:
Good point!